docs/setup/keadm.md

```diff
@@ -138,6 +138,28 @@ KubeEdge edgecore is running, For logs visit:  /var/log/kubeedge/edgecore.log
 
 6. Restart all the cloudcore and edgecore.
 
+### Support Metrics-server in Cloud
+1. The realization of this function point reuses cloudstream and edgestream modules. So you also need to perform all steps of *Enable `kubectl logs` Feature*.
+
+2. Since the kubelet ports of edge nodes and cloud nodes are not the same, the current release version of metrics-server does not support automatic port identification, so you need to manually compile the image yourself.
```

Comment:
> the current release version of metrics-server does not support automatic port identification

Better to point out exact version numbers of metrics-server that support and doesn't support

Reply:
Updated. Now the metrics-server is 0.3.7. This feature is estimated to be launched in 0.4.0. Some features of 0.4.0 are still in the early stages of development.